tests/lib/mpsc_pbuf/testcase.yaml

```diff
@@ -5,3 +5,14 @@ tests:
       qemu_arc_em qemu_arc_hs qemu_cortex_a53 qemu_cortex_m0 qemu_cortex_m3
       qemu_cortex_r5 qemu_leon3 qemu_nios2 qemu_riscv32 qemu_riscv64 qemu_x86
       qemu_x86_64 qemu_xtensa
+    integration_platforms:
+      - native_posix
+
+  lib.mpsc_pbuf_concurrent:
+    tags: mpsc_pbuf
+    platform_allow: >
+      qemu_cortex_m3 qemu_x86
```

Comment:
Maybe add a SMP platform here, sub x86_64 for x86, for example?

Reply:
add and luckily it revealed one bug.